### ISSUE TITLE
Configure idea.gdsl file

### ISSUE THREAD

**Issue by cmoulliard** [ch007m/jenkins-job-dsl#6]
## Feature request

The existing `idea.gdsl` dont work when the groovy scripts are located under `src/main/resources` and has been designed to work only with Job DSL and not Pipeline DSL under IntelliJ

The file should be enhanced to support Job & Pipeline DSL and find classes in order to allow to use `auto-completion`

Interesting links: 
- http://vgaidarji.me/blog/2018/07/30/working-with-jenkinsfile-in-intellij-idea/
- https://st-g.de/2016/08/jenkins-pipeline-autocompletion-in-intellij

**Comment by yharish991** [comment]
@cmoulliard autocompletion seems to be working without having to do anything

![image](https://user-images.githubusercontent.com/8673639/104160771-acecef80-53bf-11eb-9799-e79f916d8fba.png)

is this what you are looking to do?

**Comment by cmoulliard** [comment]
> is this what you are looking to do?

Yes but I want to be able to use it within a pipeline where this is not the case. Example : https://github.com/ch007m/jenkins-job-dsl/blob/main/src/main/resources/pipelineCheckBomDependencies.groovy

The pipeline property is defined within the file `jenkinsPipeline.gdsl
file:///Users/cmoullia/code/ch007m/jenkins-jobs-dsl/src/jenkinsPipeline.gdsl` 

See: `property(name: 'pipeline', type: 'org.jenkinsci.plugins.pipeline.modeldefinition.ModelStepLoader')` 

and dependency containing declared declared 
```
[INFO] +- org.jenkinsci.plugins:pipeline-model-definition:jar:1.7.2:test
[INFO] |  +- org.jenkinsci.plugins:pipeline-model-api:jar:1.7.2:test
[INFO] |  |  \- com.github.fge:json-schema-validator:jar:2.0.4:test
[INFO] |  |     +- com.github.fge:json-schema-core:jar:1.0.4:test
[INFO] |  |     |  \- org.mozilla:rhino:jar:1.7R4:test
[INFO] |  |     +- javax.mail:mailapi:jar:1.4.3:test
[INFO] |  |     \- com.googlecode.libphonenumber:libphonenumber:jar:5.3:test
``` 
but autocompletion dont work.

@yharish991
